dataactvalidator/config/sqlrules/c9_award_financial.sql

```diff
@@ -1,14 +1,14 @@
 SELECT afa.row_number, afa.fain, afa.uri
 FROM award_financial_assistance AS afa
 WHERE afa.submission_id = {0}
-	AND afa.federal_action_obligation > 0
+	AND CAST(afa.federal_action_obligation as numeric) > 0
```

Comment:
That makes sense. However, if we CAST a string to numeric, that will throw a SQL error, which presumably won't show up in a nice way on the error report. Again, I'm fine leaving this but think it merits more discussion. Will add to the question list.